airbyte-commons-worker/src/main/java/io/airbyte/workers/WorkerConfigs.java

```diff
@@ -109,7 +120,12 @@ public static WorkerConfigs buildCheckWorkerConfigs(final Configs configs) {
         nodeSelectors,
         configs.getUseCustomKubeNodeSelector() ? Optional.of(configs.getIsolatedJobKubeNodeSelectors()) : Optional.empty(),
         annotations,
+<<<<<<< HEAD
+        null,
+        configs.getJobKubeMainContainerImagePullSecret(),
+=======
         configs.getJobKubeMainContainerImagePullSecrets(),
```

Comment:
configs.getJobKubeMainContainerImagePullSecrets(),
```suggestion
        configs.getJobKubeMainContainerImagePullSecrets(),
```